frontend/src/views/DailyView/DatePicker.jsx

```diff
@@ -0,0 +1,58 @@
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import AdapterMoment from '@mui/lab/AdapterMoment';
+import DesktopDatePicker from '@mui/lab/DesktopDatePicker';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import React from 'react';
+
+const DatePicker = () => {
+  const [datePickerValue, setDatePickerValue] = React.useState(new Date());
+
+  const forward = () => {
+    if (datePickerValue < new Date(new Date().setDate(new Date().getDate() + 6))) {
+      setDatePickerValue(() => new Date(new Date().setDate(datePickerValue.getDate() + 1)));
+    } else {
+      alert("You can't go further");
+      setDatePickerValue(() => new Date(new Date().setDate(new Date().getDate() + 7)));
+    }
+  };
```

Comment:
Since we have date-fns installed we can make it more readable:
```jsx
  const goOneDayForward = () => {
    if (datePickerValue < addDays(new Date(), 6)) {
      setDatePickerValue(() => new Date(addDays(datePickerValue, 1)));
    } else {
      alert("You can't go further");
    }
  };
 ```